File: VB/Create a Report Bound to XPObjectSource/ORMDataModel1Code/Categories.vb
Imports DevExpress.Xpo
Imports DevExpress.Xpo.Metadata
Imports DevExpress.Data.Filtering
Imports System.ComponentModel

Namespace Create_a_Report_Bound_to_XPObjectSource.nwind

    Public Partial Class Categories

        Public Sub New(ByVal session As Session)
            MyBase.New(session)
        End Sub

        Public Overrides Sub AfterConstruction()
            MyBase.AfterConstruction()
        End Sub
    End Class
End Namespace

File: CS/ORMDataModel1Code/Categories.cs
using System;
using DevExpress.Xpo;
using DevExpress.Xpo.Metadata;
using DevExpress.Data.Filtering;
using System.Collections.Generic;
using System.ComponentModel;
using System.Reflection;
namespace Create_a_Report_Bound_to_XPObjectSource.nwind {

    public partial class Categories {
        public Categories(Session session) : base(session) { }
        public override void AfterConstruction() { base.AfterConstruction(); }
    }

}
